spec/tests.rb

```diff
@@ -0,0 +1,94 @@
+require_relative '/Users/Sunzi/Desktop/CarCleaningService/Service.rb'
+
+describe 'one car' do
+  subject {service = Service.new}
+  
+  context 'when the car needs to be picked up today' do
+    before {allow(Time).to receive(:now).and_return(Time.new(2021,04,20,14,30))}
```

Comment:
I would add a little bit of spacing here:
```ruby
before { allow(Time).to receive(:now).and_return(Time.new(2021, 04, 20, 14, 30)) }
```

Same can apply for multiple tests inside this file also